docs/clients/brushless_drive.rst

```diff
@@ -105,72 +105,116 @@ Message Table
+| 54     | motoring_limit_adjust         | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 55     | regen_limit_kp                | get, set, save | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 56     | regen_limit_ki                | get, set, save | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 57     | regen_limit_max               | get, set, save | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 58     | motoring_limit_kp             | get, set, save | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 59     | motoring_limit_ki             | get, set, save | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 60     | motoring_limit_max            | get, set, save | float     |          |                                                                                                                                      |      |
```

Comment:
Description: "Maximum amount of drive voltage adjustment the closed loop portion of the motoring dynamic supply current limiter will apply to the calculated voltage limit."

Units are V